src/Symfony/Component/VarDumper/Tests/Caster/IntlCasterTest.php

```diff
@@ -294,4 +294,9 @@ public function testCastDateFormatter()
 EOTXT;
         $this->assertDumpEquals($expected, $var);
     }
+
+    private function normalizeU202F(string $input): string
```

Comment:
```suggestion
    private function normalizeNarrowNoBreakSpaceCharacter(string $input): string
```

Reply:
Thank's, I didn't understand that I was able to directly commit your suggestion so I did a separate commit